OPass/Storyboard/Main.storyboard

```diff
@@ -457,7 +455,7 @@ Please copy and paste the token into next field or click the token link in the r
                         <outlet property="ivUserPhoto" destination="01s-25-wxe" id="rr3-07-Jkv"/>
                         <outlet property="lbHi" destination="L3Y-MS-x0I" id="uJH-0a-vrQ"/>
                         <outlet property="lbUserName" destination="vJH-ug-pK0" id="se2-C1-fIo"/>
-                        <segue destination="VKq-4v-Yvp" kind="custom" identifier="ShowGuide" customClass="EmbeddedNavigationTabBarControllerSegue" customModule="OPass" customModuleProvider="target" id="lBg-aP-1xi"/>
+                        <segue destination="VKq-4v-Yvp" kind="custom" identifier="ShowGuide" customClass="EmbeddedNavigationControllerSegue" customModule="OPass" customModuleProvider="target" id="lBg-aP-1xi"/>
```

Comment:
Why this class was rollbacked to `EmbeddedNavigationControllerSegue` since it was fix by following ?

https://github.com/CCIP-App/CCIP-iOS/blob/2218044f99d86bf67878e0ae29034b03b6617e8a/OPass/Base.lproj/Main.storyboard#L460